### PR TITLE
try it update

### PR DESCRIPTION
**Pull request by divyaamunugama**
Remaining tasks: 
Update the diagram

### CODE REVIEW

File: en/docs/guides/organization-login/try-organization-login.md [hunk 1]
```diff
@@ -1,114 +1,169 @@
...
-    !!! note
-        Be sure to select **Management Application** when creating the application.
-        
-        <img src="../../../assets/img/guides/organization-login/create-business-applications/management_application.png" alt="Management Application" width="300" style="border:1px solid grey">
+To configure the business applications:
 
-3.  Replace the consumerKey and consumerSecret values with the OAuth client key and OAuth client secret values that were generated for the newly created application.
+1. On the CIAM Private Cloud console, go to **Develop > Applications**.
+2. Click **+ New Application** and select **Standard-Based Application**.
+3. Enter `Guardio-SaaS-app` as the application name and select `OAuth2 / OpenID Connect` as the **Protocol**.
```
Comment (Sachin-Mamoru) [inline en/docs/guides/organization-login/try-organization-login.md:52]:
```suggestion
3. Enter `Guardio-SaaS-app` as the application name and select `OAuth2.0 / OpenID Connect` as the **Protocol**.
```

File: en/docs/guides/organization-login/try-organization-login.md [hunk 2]
```diff
@@ -1,114 +1,169 @@
...
+<img src="../../../assets/img/guides/organization-login/try-it-out/share_app_with_sub_orgs.png" alt="Share App with suborganizations" width="600" style="border:1px solid grey">
+
+## Step 4: Configure the Sign-in method
+
+After you share the application with the sub-organizations, an **Organization SSO** IdP named `Organization Login` will be automatically created and assigned as a sign-in method for the application.
+
+To check if the IdP is assigned to the application:
+
+1. On the CIAM Private Cloud console, go to **Develop > Applications** and select `Guardio-SaaS-App`.
+2. On the **Sign-in Method** tab, check if the **Organization Login** IdP has been assigned.
```
Comment (Sachin-Mamoru) [inline en/docs/guides/organization-login/try-organization-login.md:82]:
```suggestion
2. On the **Sign-in Method** tab, check if the generated **Organization Login** IdP has been assigned.
```

**Review by Sachin-Mamoru — APPROVED** [review]

File: en/docs/guides/organization-login/try-organization-login.md [hunk 3]
```diff
@@ -1,114 +1,170 @@
-# Try organization login
+# Try Organizational Login Flow
 
 Let's try organization login using a sample scenario.
 
 ## Scenario
 
-**Guardio** is an auto insurance service company that has many employees who use different credentials to sign in to different internal organizational applications. **Guardio-SaaS-App** is one such applicaiton. Here, Guardio Insurance is using the WSO2 Identity Server B2B Private CIAM Solution, therefore it’s configured as the super organization.
+**Guardio** is an auto insurance service company that has many employees who use different credentials to sign in to many internal applications.  To mange logins to its many applications, Guardio is using the WSO2 Identity Server B2B Private CIAM Solution where Guardio is configured as a super organization. *Guardio-SaaS-App** is one such application.
```
Comment (Nashaath) [inline en/docs/guides/organization-login/try-organization-login.md:7]:
```suggestion
**Guardio** is an auto insurance service company that has many employees who use different credentials to sign in to many internal applications.  To manage logins to its many applications, Guardio uses the WSO2 Identity Server B2B Private CIAM Solution where Guardio is configured as a super organization. *Guardio-SaaS-App** is one such application. 
```

File: en/docs/guides/organization-login/try-organization-login.md [hunk 4]
```diff
@@ -153,43 +209,31 @@ For the Car Traders organization, [create the user](../../org-user-management) S
...
 
-    <img src="../../../assets/img/guides/organization-login/try-it-out/logged_in_to_sample_app.png" alt="Logged in to Sample App" width="700" style="border:1px solid grey">
+    <img src="../../../assets/img/guides/organization-login/try-it-out/sample_app_org.png" alt="Sample App Orgs" width="700" style="border:0px solid grey">
 
-7.  Now, select Best Auto Mart Organization and try to log in. You’ll be redirected to the Asgardeo IdP based login page.
+5. Enter the user credentials and click **Continue** to log in to the application.
+    You will be successfully logged in to the sample application.
 
-    <img src="../../../assets/img/guides/organization-login/try-it-out/asgardeo_based_login.png" alt="Asgardeo based Login" width="400" style="border:1px solid grey">
+6. Logout of the application, select `Best Auto Mart` organization and try to log in. You will be redirected to the Asgardeo login page.
```
Comment (Nashaath) [inline en/docs/guides/organization-login/try-organization-login.md:239]: If we are going with the terms "sign in" and "sign out", we should consider using the same throughout the doc.

**Review by Nashaath — APPROVED** [review]

**Pull request merged**